src/widget/loginscreen.cpp

```diff
@@ -31,6 +31,16 @@
 #include <QMessageBox>
 #include <QDebug>
 
+#ifdef Q_OS_WIN32
+    #include <windows.h>
+#else
+    #include <X11/XKBlib.h>
```

Comment:
@Diadlo There's no real need to do it, since qTox already "depends" on a few xorg-only things – what needs to be done, is to properly `define` things.